src/app/components/client/UserMenu.tsx

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { Session } from "next-auth";
+import { signIn, signOut } from "next-auth/react";
+
+export type Props = {
+  session: Session | null;
+};
+
+export const UserMenu = (props: Props) => {
+  // Placeholder to enable login/logout until we create an actual menu
+  if (!props.session) {
+    return <button onClick={() => signIn("fxa")}>Sign in</button>;
```

Comment:
nit: all feels very placeholdery, but not sure if we want to localize these buttons now or not. 🤷